src/Parallel/Main.hpp

```diff
@@ -112,6 +112,16 @@ class Main : public CBase_Main<Metavariables> {
                                    funcl::Identity, std::index_sequence<>>>
           reduction_data);
 
+  /// Add an exception to the list of exceptions. Upon a phase change we print
+  /// all the received exceptions and exit.
+  ///
+  /// Upon receiving an exception all algorithms are terminated to guarantee
+  /// quiescence occurs soon after the exception is reported. In the future we
+  /// will want to change to a "cleanup"-type phase that allows each distributed
+  /// object to do some sort of cleanup, e.g. writing diagnostics to disk,
+  /// before shutdown.
```

Comment:
This is done now, yes?

Reply:
Yes, since it's happening in the same PR I'll just completely remove that part of the comment.